bpf/lib/common.h

```diff
@@ -356,6 +356,11 @@ ipv6_ct_tuple_get_daddr(struct ipv6_ct_tuple *tuple)
 #endif
 }
 
+static inline bool ipv6_is_dns(struct ipv6_ct_tuple *tuple)
+{
+	return tuple->nexthdr == IPPROTO_UDP && tuple->dport == bpf_htons(53);
```

Comment:
Is the sport/dport reversed here? I assume you want to match on the sport being 53 to catch the DNS response with the A and AAAA record.

Reply:
Yes, it's reversed. I cam add a comment to explain.